.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "libswiftnav"]
 	path = libswiftnav
-	url = https://github.com/swift-nav/libswiftnav
+	url = https://github.com/adel-mamin/libswiftnav
```

Comment:
Yes, this PR is contingent on https://github.com/swift-nav/libswiftnav/pull/315. I've made an explicit not about that in the PR comment. The dependency on https://github.com/swift-nav/libswiftnav/pull/315 is due to L2C correlator, which is part of libswiftnav. https://github.com/swift-nav/libswiftnav/pull/315 was created for libswiftnav/more_peregrine_updates. Once libswiftnav/more_peregrine_updates is merged, I will rebase https://github.com/swift-nav/libswiftnav/pull/315 on top of libswiftnav/main.